src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -208,7 +210,7 @@ public static ClassModel parseFileContents(String filePath) {
                 }
 
                 // gather up our comments
-                if (strLine.startsWith("/*")) {
+                if (strLine.startsWith("/**")) {
```

Comment:
Yes, this project was resurrected from the dead so we could fully document the Salesforce Foundation's Nonprofit Starter Pack.  Code is here on gitHub: https://github.com/SalesforceFoundation/Cumulus.  The current ApexDoc output was published here to give you an idea of what it looks like: http://developer.salesforcefoundation.org/Cumulus/ApexDocumentation/

note that we haven't finished documenting all of our classes.